Test/Unit/ViewModel/HyvaGridViewModelTest.php

```diff
@@ -109,6 +110,16 @@ private function createStubMassActionFactory(): MockObject
         return $this->createMock(MassActionInterfaceFactory::class);
     }
 
+    /**
+     * @return MockObject|HyvaPrefetchEventDispatcher
+     */
+    private function createStubHyvaPrefetchEventDispatcher(array $columnDefinitions): MockObject
+    {
+        $column = $this->createMock(HyvaPrefetchEventDispatcher::class);
```

Comment:
Please rename `$column` to `$dispatcher`.

Instead of passing the array of $columnDefinitions to this method and declaring that as the return value, it could be simplified with
```php
$dispatcher->method('dispatch')->willReturnArgument(0);
```